replica/backup.go

```diff
@@ -54,6 +62,11 @@ func (rr *RestoreStatus) UpdateRestoreStatus(snapshot string, rp int, re error)
 	rr.Progress = rp
 	rr.Error = re
 	rr.LastUpdatedAt = time.Now()
+	if rr.Progress == 100 {
+		rr.State = StateComplete
+	} else if rr.Error != nil {
```

Comment:
Just noticed the error should be a string too for RestoreStatus.

Also, it's error state then, not incomplete.